source/braille.py

```diff
@@ -2499,7 +2503,18 @@ def _handlePendingUpdate(self):
 			scrollTo: Optional[TextInfoRegion] = None
 			self.mainBuffer.saveWindow()
 			for region in self._regionsPendingUpdate:
-				region.update()
+				from treeInterceptorHandler import TreeInterceptor
+				if isinstance(region.obj, TreeInterceptor) and not region.obj.isAlive:
+					log.debugWarning("Skipping region update for died tree interceptor")

```

Comment:
Perhaps this should just be a debug level log if this is an expected path.

```suggestion
					log.debug("Skipping region update for died tree interceptor")
```